mysql/datadog_checks/mysql/statements.py

```diff
@@ -249,8 +253,45 @@ def _query_summary_per_statement(self):
             {}
             """.format(condition)
 
+        if collect_prepared_statements:
+            prepared_sql_statement_summary = """\
+                SELECT  `owner_object_schema` AS `schema_name`,
+                        NULL AS `digest`,
+                        `sql_text` AS `digest_text`,
+                        `count_execute` AS `count_star`,
+                        `sum_timer_execute` AS `sum_timer_wait`,
+                        `sum_lock_time`,
+                        `sum_errors`,
+                        `sum_rows_affected`,
+                        `sum_rows_sent`,
+                        `sum_rows_examined`,
+                        `sum_select_scan`,
+                        `sum_select_full_join`,
+                        `sum_no_index_used`,
+                        `sum_no_good_index_used`,
+                        NOW() AS `last_seen`
+                FROM performance_schema.prepared_statements_instances
+                WHERE `sql_text` NOT LIKE 'EXPLAIN %' OR `sql_text` IS NULL
+                """
+
+            sql_statement_summary = f"""\
+                {sql_statement_summary}
+                UNION ALL
+                {prepared_sql_statement_summary}
+                """
+        if not only_query_recent_statements:
+            sql_statement_summary = f"""\
+                {sql_statement_summary}
+                ORDER BY `count_star` DESC
+                LIMIT 10000
+                """
+
         with closing(self._get_db_connection().cursor(CommenterDictCursor)) as cursor:
-            args = [self._last_seen] if only_query_recent_statements else None
+            if only_query_recent_statements:
+                arg_count = 1 + (1 if collect_prepared_statements else 0)
```

Comment:
I'm not following why we need additional arg count

Reply:
We don't - removed now.